core/templates/dev/head/filters/string-utility-filters/format-timer.pipe.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview FormatTimer filter for Oppia.
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({name: 'formatTimer'})
+export class FormatTimerPipe implements PipeTransform {
+  transform(input: any): any {
```

Comment:
This doesn't look like really `any`, please verify!

Reply:
Done